target-app/Home/Views/HomeViewController.swift

```diff
@@ -62,6 +69,23 @@ class HomeViewController: UIViewController, MKMapViewDelegate {
         if let coordinates = mapView.userLocation.location?.coordinate {
             mapView.setCenter(coordinates, animated: true)
         }
+        
+        let longTapGesture = UILongPressGestureRecognizer(target: self, action: #selector(longTap))
+        mapView.addGestureRecognizer(longTapGesture)
+    }
+    
+    // MARK: - ACTIONS
+    
+    @objc func longTap(sender: UIGestureRecognizer){
```

Comment:
Is this really needed? I don't think the user should be able to select the map location to create a target. Targets are created from the users' current location.